cli/validate.go

```diff
@@ -58,16 +70,16 @@ func validateRun(cmd *cobra.Command, args []string) {
 	for _, request := range requests {
 		ok, err := controller.ValidateRequest(request)
 		if ok {
-			newLogger.Info(newCtx, "Group '%v' is valid.", request.Group)
+			fmt.Printf("Group '%v' is valid.\n", request.Group)
```

Comment:
this reverses what Joe just added. I personally like it though - but is it intended? :dancer: